api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/sms/NexmoProperties.java

```diff
@@ -35,5 +35,10 @@
     /**
      * The nexmo application id.
      */
-    private String applicationId = "CAS";
+    private String applicationId;
+
+    /**
+     * Private key content path to sign Nexmo messages
+     */
+    private String privateKeyContentsPath;
```

Comment:
This doesn't make much sense. Are we expected to put in contents of something, or is this a path to something/file? It should either be `private String privateKeyContentsPath;` or ` private String privateKeyPath;`. If it's a path, it's best to define it as a Resource, and please document the setting and explain when and how it might be used in combination of other settings.

Reply:
My bad, private key is for phone call. I'll add secret signature for sms